### PR TITLE
Added Inter Font from @mattro

### PR DESCRIPTION
**Pull request by alessandrofavero**
Added Inter font to the app as a global font.

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -101,7 +101,7 @@ packages:
       name: pedantic
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.9.0"
+    version: "1.8.0+1"
```
Comment (maks) [inline pubspec.lock]: @alessandrofavero not sure why this version is being downgraded? 1.9 is latest in pub.dev - are you on latest flutter stable channel?

**Review by maks — CHANGES_REQUESTED** [review]
just one small nit about pubspec downgrading pendatic package

File: pubspec.lock (hunk 1 above)
Comment (alessandrofavero) [reply pubspec.lock]: Yes I am. That's weird I didn't touch pubspec.yaml other than adding the font. I'll fix it.

Comment (alessandrofavero) [reply pubspec.lock]: All fixed in the latest commit.

**Review by maks — APPROVED** [review]
:+1:

**Pull request merged**